app/src/main/java/nextstep/payments/ui/main/PopulatedPaymentCardScreen.kt

```diff
@@ -0,0 +1,69 @@
+package nextstep.payments.ui.main
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import nextstep.payments.data.Card
+import nextstep.payments.ui.component.MaskedCardNumberText
+import nextstep.payments.ui.component.PaymentCard
+
+@Composable
+fun PopulatedPaymentCard(
+    card: Card,
+    modifier: Modifier = Modifier
+) {
+    Box(
+        modifier = modifier
+            .width(208.dp)
+            .height(124.dp)
+    ) {
+        PaymentCard()
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .fillMaxHeight()
```

Comment:
```suggestion
                .fillMaxSize()
```
너비와 높이를 전체를 채우시는 것이라면 fillMaxSize를 사용하셔도 됩니다.

Reply:
컴포즈 모르는거 너무 티가 나네요🥶